exercises/e2.js

```diff
@@ -41,10 +37,9 @@ console.log(room1, room2, room3);
 
 // Your code goes here...
 function totalRoomsArea(area1, area2, area3) {
-  return area1 + area2 + area3;
+  return (area1) + (area2) + (area3);
 }
-var roomsTotal = (24, 35, 24);
-console.log(roomsTotal);
+
 
 
 // Make the function call in the parentheses of the console.log statement.
```

Comment:
Make the function call in the parentheses of the console.log statement . 
use room1, room2, room3 variables as arguments for the function call.
very confused on this . Id assume it would be 
const result = function () {
   console.log(result( ));
}
I am aware this is incorrect after failed attempts..
original code is 
const result = function ( ) {
   console.log(/* replace this comment with a function call here... */);
}
was under the assumption that the function call to be placed within the console. parentheses is result( ).

Then for variables room1, room2, room3 they are already declared above in the same exercise as 
var room1 = rectArea(4, 6);
var room2 = rectArea(13, 3);
var room3 = rectArea(4, 8);
so I used them as my arguments or I think I did lol . 
I'm sure it is way easier than I exaggerate lol.
not sure what is pre declared in some of the functions within the test . 
a lot of failed attempts on this question particularly.